pkgs/applications/audio/wolf-spectrum/default.nix

```
@@ -0,0 +1,50 @@
+{ pkgs
+, pkgconfig
+, libjack2
+, lv2
+, xorg
+, liblo
+, libGL
+}:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "wolf-spectrum";
+  version = "1.0.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "pdesaulniers";
+    repo = "wolf-spectrum";
+    rev = "v${version}";
+    sha256 = "17db1jlj7vb1xyvkdhhrsvdbwb7jqw6i4168cdvlj3yvn2ra8gpm";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = with pkgs; [ pkgconfig ];
```

canonical name over alias
```suggestion
  nativeBuildInputs = with pkgs; [ pkg-config ];
```